README.md

```diff
@@ -40,103 +42,117 @@ We will use the same project structure we used for the previous project. Classes
-- As an administrator, I can check whether a given block has any rooms available
-- As an administrator, I can reserve a room from within a block of rooms
+- As a user of the hotel system,
+  - I can create a Hotel Block if I give a date range, collection of rooms, and a discounted room rate
+  - I can see an exception raised if I try to create a Hotel Block and at least one of the rooms is unavailable for the given date range
+  - Given a specific date, and that a room is set aside in a hotel block for that specific date, I cannot reserve that specific room for that specific date, because it is unavailable
+  - Given a specific date, and that a room is set aside in a hotel block for that specific date, I cannot create another hotel block that includes that specific room for that specific date, because it is unavailable
+  - I can check whether a given block has any rooms available
+  - I can reserve a room from within a block of rooms
 
-### Constraints
+#### Details
 
 - A block can contain a maximum of 5 rooms
 - When a room is reserved from a block of rooms, the reservation dates will always match the date range of the block
 - All of the availability checking logic from Wave 2 should now respect room blocks as well as individual reservations
 
-## Before Submissions
+## Non-Functional Requirements
 
-Usually by the end of a project, we can look back on what we made with a clearer understanding of what we actually needed. In industry, this is a great time to do a refactor of some sort. For this project however, you're off the hook... for the moment. We will be revisiting our hotels later on on the course, and you may want to make some changes at that point. 
+### Testing Requirements
 
-- Create a new file in the project called `refactors.txt`
-- Make a short list of the changes that you could make, particularly in terms of naming conventions
-- These notes will be used by you in a few weeks, so make sure that they are detailed enough that someone else could understand your thinking and follow your directions
-- **Do not make any further changes to your code at this time**
+- Utilize the spec helper file
+- Run tests automatically whenever files are added or changed using the `guard` command
+- The final project submission should have **95% code coverage** using `simplecov`
 
 ## Optional Enhancements
 
-You should not be working on these (or even thinking about them) until you have fully completed wave 3.
+You should not be working on these (or even thinking about them!) until you have fully completed wave 3.
+
+<details>
+  - Add functionality that allows for setting different rates for different rooms
+  - Read and write CSV files for each piece of data that your system is storing
+  - Create a CLI to interact with your booking system. Put all of this code in a file `main.rb` that is separate from your `lib` code. Working on this optional enhancement should not break the other requirements of this project
```

Comment:
You need an empty line between `<details>` and the list, otherwise it does not render correctly.